aws/src/database/rds/addresses/session_for_addresses.py

```diff
@@ -0,0 +1,52 @@
+"""
+Note: Ensure to install unixodbc to be able to use pyodbc
+"""
+
+from sqlalchemy import create_engine, text
+from sqlalchemy.orm import sessionmaker, Session as SA_Session
+from psycopg2 import connect as psycopg2_connect
+
+from mypy_boto3_ssm import SSMClient
+
+from aws.src.authentication.generate_aws_resource import generate_aws_service
+from enums.enums import Stage
+
+from entities.HackneyAddress import HackneyAddress, Base as HackneyAddressBase
+
+
+def session_for_addresses(stage: Stage, expire_on_commit=True, local_port=1433) -> sessionmaker[SA_Session]:
+    """
+    Connect to addresses database on a local port
+    :param stage: Stage to connect to
+    :param expire_on_commit: Select to persist detached objects after transaction commit
+    :return: SQLAlchemy sessionmaker - to be used as a context manager
+    """
+
+    pg_username_path = f"/addresses-api/{stage.to_env_name()}/postgres-username"
```

Comment:
Add a space at the beginning so it works for windows machine. Something like:
`" /addresses-api/{stage.to_env_name()}/postgres-username"`

Reply:
It should be fine in the Python script as that's not parsed by Powershell / Git Bash